composer.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "salsadigital/migrate",
-  "description": "OpenSource migraiton tool for Drupal 8",
+  "name": "salsadigital/merlin-framework",
```

Comment:
is name "salsadigital" or "salsadigitalorgau"?
in Readme it uses `salsadigitalorgau/merlin-framework` but if name in composer.json is `salsadigital/merlin-framework` installation will not work.
also readme uses version ^1.0 - which doesn't exist, so that will fail too
maybe use a placeholder XX instead of a version

anyway, thanks for prompt response 🥇

Reply:
Nice catch @jannakha (again)! Updated package name to match org/repo and tweaked README to suit.

Thanks for raising these, we'll get a point release out shortly to resolve. 🙏